src/processor/map/map_index_scan_node.cpp

```diff
@@ -20,15 +20,21 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapIndexLookup(LogicalOperator* lo
         auto nodeTable = storageManager->getTable(info.nodeTableID)->ptrCast<storage::NodeTable>();
         auto offsetPos = DataPos(outSchema->getExpressionPos(*info.offset));
         auto keyPos = DataPos(outSchema->getExpressionPos(*info.key));
-        indexLookupInfos.push_back(std::make_unique<IndexLookupInfo>(nodeTable, keyPos, offsetPos));
+        std::vector<DataPos> warningDataPos;
+        for (const auto& warningDataExpr : info.warningData) {
+            warningDataPos.push_back(DataPos(outSchema->getExpressionPos(*warningDataExpr)));
```

Comment:
we have a `getDataPos` function I believe.

Reply:
updated